control-plane/roles/monitoring/templates/metrics-exporters/metal-metrics-exporter.yaml

```diff
@@ -63,3 +63,20 @@ spec:
             runAsUser: 999
             allowPrivilegeEscalation: false
             runAsNonRoot: true
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    release: kube-prometheus-stack
+  name: metal-metrics-exporter
+spec:
+  endpoints:
+    - path: /metrics
+      port: metrics
+      interval: 60s
+      scrapeTimeout: 30s
+  selector:
+    matchLabels:
+      app: metal-metrics-exporter
+
```

Comment:
I think we have this already deployed in `control-plane/roles/monitoring/templates/servicemonitors/metal-api.yaml`.